docs/index.md

```diff
@@ -7,77 +7,140 @@
 
-## Due to the number of downloads I am seeing on pypistats.org, I am including these instructions in case a beta update breaks something on your end:
+## Library Reliability Testing
 
-### Emergency Rollback Instructions
+[Format Compliance Reliability Testing](reliability_testing/format_compliance.md)
 
-If you experience issues with version 0.1.0b4, you can roll back to the previous stable version (0.1.0b3.post3) using one of these methods:
+[Behavioral Testing Reliability](reliability_testing/behavioral_testing_reliability.md)
 
-#### Method 1: Direct Installation of Previous Version
+[Behavioral Testing Non-determinism At Semantic Boundary - Discovery and Analysis](reliability_testing/behavior_at_semantic_boundaries.md)
 
-```
-pip uninstall llm-app-test 
-pip install llm-app-test==0.1.0b3.post3
-```
-#### Method 2: Force Reinstall (if Method 1 fails)
+## Testing Philosophy
 
-```
-pip install --force-reinstall llm-app-test==0.1.0b3.post3
-```
-#### Verification
-After rolling back, verify the installation:
-```
-import llm_app_test 
-print(llm_app_test.version) # Should show 0.1.0b3.post3
-```
+When integrating LLMs into your application, treat them as you would any closed-source third-party library:
 
-## Need testing ideas? Check out the tests we used to test llm-app-test [here](https://github.com/Shredmetal/llmtest/tree/main/tests)
+1. Write tests for expected behaviour
+2. Focus on interface boundaries
+3. Test application-level functionality
+4. Maintain clear separation of concerns
+
+### ⚠️ Important Information on Understanding Responsibilities
+
+This library is built by software engineers to give software engineers a tool to validate the behaviour of applications that have had an LLM stuffed in them. It is **NOT** a Data Science tool nor a replacement for model metrics used by Data Science teams to validate model suitability.
+
+#### Software Engineer's Role
+
+- Write tests for expected application behaviour
+- Validate inputs and outputs
+- Ensure proper integration
+- Monitor system performance
+- Escalate consistent failures to DS team (as this might indicate a fundamental problem with the model, or perhaps to seek assistance with the `expected_behavior` prompt in the `assert_behavioral_match` function)
+
+#### Data Science Team's Role
+
+- Handle model-level issues
+- Address consistent test failures
+- Evaluate model suitability
+- Optimise model performance
+- Adjust prompts when needed
+
+### When to Escalate
+
+Escalate to your Data Science team when:
+
+1. Tests consistently fail despite correct implementation
+2. Model responses are consistently inappropriate
+3. Performance degradation is observed
+4. Pattern of failures indicates model-level issues
+
+### 🔍 What Makes This Different?
+
+This is an ENGINEERING tool, not a data science tool. The difference is crucial:
+
+Data Science Tools:
+- Test model performance
+- Evaluate model accuracy
+- Measure model metrics
+
+llm_app_test (Engineering Tool):
```

Comment:
llm-app-test

Reply:
Undone - standardise to llm_app_test because that's what the import statement is going to say. llm-app-test is only used for pip install because PyPI converts underscores to hyphens